project/src/engines/hoomd/project.py

```diff
@@ -32,34 +32,60 @@ def run_hoomd(job):
     structure = ff.apply(filled_box)
 
     write_gsd(structure, job.fn("init.gsd"), ref_distance=rd, ref_energy=re)
-    # NOTE: create_hoomd3_forcefield depends on PR
-    # https://github.com/mosdef-hub/mbuild/pull/871
+    # ref_distance: angstrom -> nm
+    # ref_energy: kcal/mol -> kJ/mol
+    # ref_mass: dalton -> amu
     snapshot, forcefield, ref_vals = create_hoomd3_forcefield(
-        structure, ref_distance=10, ref_energy=1 / 4.184
+        structure, ref_distance=0.1, ref_energy=1 / 4.184, ref_mass=0.9999938574
     )
 
     device = hoomd.device.auto_select()
-    simulation = hoomd.Simulation(device=device, seed=job.sp.replica)
-    simulation.create_state_from_snapshot(snapshot)
+    sim = hoomd.Simulation(device=device, seed=job.sp.replica)
+    sim.create_state_from_snapshot(snapshot)
     gsd_writer = hoomd.write.GSD(
-        filename=job.fn("traj.gsd"),
+        filename=job.fn("trajectory.gsd"),
         trigger=hoomd.trigger.Periodic(10000),
         mode="ab",
     )
-    simulation.operations.writers.append(gsd_writer)
+    sim.operations.writers.append(gsd_writer)
+
+    logger = hoomd.logging.Logger(categories=["scalar"])
+    logger.add(sim, quantities=["timestep", "tps"])
+    thermo_props = hoomd.md.compute.ThermodynamicQuantities(
+        filter=hoomd.filter.All()
+    )
+    sim.operations.computes.append(thermo_props)
+    logger.add(
+        thermo_props,
+        quantities=[
+            "kinetic_energy",
+            "potential_energy",
+            "pressure",
+            "kinetic_temperature",
+            "volume",
+        ],
+    )
+    file = open("log.txt", mode="x", newline="\n")
+    table_file = hoomd.write.Table(
```

Comment:
Was there a strong preference for the table file over a gsd to log values?

Reply:
If we're not totally set on table files, I would vote for GSD files because:
1. They are _not_ human-readable, which forces you to process them programmatically 
2. You don't really have to worry about headers and stuff, the values are stored in the file with descriptive field names
3. They are much more space-efficient than text files (although that's probably not an issue for this particular project)

Reply:
I am going off [this reference card](https://github.com/mosdef-hub/reproducibility_study/projects/1#card-66677615) from the project board. We chose not to combine them in one file since the write out periods would not be the same. I don't have a strong preference, but it is nice that the values are human readable.

Reply:
We can write to multiple GSD files, one for the trajectory and one more frequently for the thermo data without trajectory data.

Reply:
All that being said, let's not let this hold us up for actually generating the data. I'm OK with text files.

Reply:
Since it will be a bit more of a challenge to convert all the other engine's thermodynamic data to a GSD compared to just the trajectory, I am a bit more in favor of our engines all writing out to something a bit more similar for this case.